nuitka/build/static_src/MainProgram.c

```diff
@@ -1367,6 +1379,81 @@ void setLANGSystemLocaleMacOS(void) {
 #endif
 
 static int Nuitka_Main(int argc, native_command_line_argument_t **argv) {
+
+#if(defined(_NUITKA_LINUX_CONSOLE_MODE_FORCE) || defined(_NUITKA_LINUX_CONSOLE_MODE_DETECT)) && !defined(_WIN32) && !defined(__APPLE__)
```

Comment:
You can check for __linux__ no problem here, if we want to add others, like the ones for FreeBSD, we can sure, but right now we seem to limit the OS anyway to Linux, and I am fine for that.